src/pages/NodesTelemetry.vue

```diff
@@ -138,121 +137,239 @@ function formatTimeSpan(date1: Date | null, date2: Date | null) {
       >
         <template #header>
           <div class="nodes-telemetry-page__list-row">
-            <span class="h-sm cell">{{ $t('telemetry.publicKey') }}</span>
             <span class="h-sm cell">{{ $t('telemetry.publicUrl') }}</span>
-            <span class="h-sm cell">{{ $t('telemetry.block') }}</span>
-            <span class="h-sm cell">{{ $t('telemetry.blockPropagationTime') }}</span>
-            <span class="h-sm cell">{{ $t('telemetry.txnsInQueue') }}</span>
+            <span class="h-sm cell">{{ $t('telemetry.connectionStatus') }}</span>
             <span class="h-sm cell">{{ $t('telemetry.location') }}</span>
+            <span class="h-sm cell">{{ $t('telemetry.publicKey') }}</span>
+            <span class="h-sm cell">{{ $t('telemetry.blocksGossiping') }}</span>
+            <span class="h-sm cell">{{ $t('telemetry.tnxsGossiping') }}</span>
+            <span class="h-sm cell">{{ $t('telemetry.block') }}</span>
+            <span class="h-sm cell">{{ $t('telemetry.commitTime') }}</span>
+            <span class="h-sm cell">{{ $t('telemetry.avgCommitTime') }}</span>
+            <span class="h-sm cell">{{ $t('telemetry.queue') }}</span>
+            <span class="h-sm cell">{{ $t('telemetry.uptime') }}</span>
           </div>
         </template>
 
         <template #row="{ item }">
           <div
-            v-if="item.metrics && item.info"
+            v-if="item.info && item.info.telemetry_unsupported"
+            class="nodes-telemetry-page__list-row_unsupported row-text cell"
+          >
+            <BaseLink :to="item.info.url">
+              {{ item.info.url }}
+            </BaseLink>
+            - {{ $t('telemetry.peerDoesntProvideTelemetry') }}
+          </div>
+          <div
+            v-else-if="item.info && item.status"
             class="nodes-telemetry-page__list-row"
           >
             <BaseHash
-              :hash="item.info.public_key"
-              type="medium"
+              :hash="item.info.url"
+              :link="item.info.url"
               copy
-              class="row-text-monospace cell"
+              class="cell"
+              type="medium"
             />
+
+            <span
+              class="row-text cell"
+              :class="[
+                item.info.connected
+                  ? 'nodes-telemetry-page__list-row-connection-success'
+                  : 'nodes-telemetry-page__list-row-connection-error',
+              ]"
```

Comment:
```suggestion
              :data-connected="item.info.connected"
```

This is less verbose. Then select in CSS by e.g. `[data-connected="true"]`